test/OrchardCore.Tests/Apis/Context/BlogPostDeploymentContext.cs

```diff
@@ -65,37 +65,32 @@ public JObject GetContentStepRecipe(ContentItem contentItem, Action<JObject> mut
 
         public async Task<HttpResponseMessage> PostRecipeAsync(JObject recipe, bool ensureSuccess = true)
         {
-            using (var zipStream = new MemoryStream())
+            using var zipStream = new MemoryStream();
+            using (var zip = new ZipArchive(zipStream, ZipArchiveMode.Create, true))
```

Comment:
We should be able to remove the extra curly bracket here

Reply:
Maybe but it would change the behavior, that's why it is not suggested, because currently it is related to a block that does not go to the end of the parent block.